reviewing/plugin-commands-licenses/test/__snapshots__/index.ts.snap

```diff
@@ -64,7 +64,7 @@ exports[`pnpm licenses: should correctly read LICENSE file with executable file
 │                              │         │ https://github.com/feross/safe-buffer                                                                  │
 ├──────────────────────────────┼─────────┼────────────────────────────────────────────────────────────────────────────────────────────────────────┤
 │ string_decoder               │ MIT     │ The string_decoder module from Node core                                                               │
-│                              │         │ https://github.com/rvagg/string_decoder                                                                │
+│                              │         │ https://github.com/nodejs/string_decoder                                                               │
```

Comment:
The behavior of this test changed, as the snapshot used an older version of the package before and now uses a newer version